### ISSUE TITLE
Ignore titles that begin with attributes for the "not capitalized" error

### ISSUE THREAD

**Issue by kilfoyle** [elastic/vale-rules#69]
Very small issue. :-)
 
It's probably not a common problem, but we have instances in the docs where a section or page title begins with an attribute. For [example](https://github.com/elastic/docs-content/blob/53556a1c54ce084231dd903eaf1fc09f4fcd24c8/manage-data/lifecycle/index-lifecycle-management.md?plain=1#L27):

```
## {{ilm-init}} availability
```

This renders correctly as "ILM availability" but is flagged by the linter as not beginning with a capital:

<img width="706" height="86" alt="Image" src="https://github.com/user-attachments/assets/75bacecb-3e3a-4c42-8e0a-032b7b18d3f5" />

Perhaps the linter should either i) exclude titles beginning with attributes from consideration, or ii) be aware of how the attributes are interpolated to check capitalization more accurately.

Thanks!
